apps/docs/pages/guides/resources/migrating-to-supabase/render.mdx

```diff
@@ -83,6 +83,13 @@ Run `pg_dump --help` for a full list of options.
 </TabPanel>
 </Tabs>
 
+<Admonition type="caution">
+
+ - If you plan on migrating a database larger than 6 GB, we recommend [contacting support](https://app.supabase.com/support/new) to ensure that you'll have the proper disk size pre-provisioned. You can read more about how the disk is managed on Supabase on the [Database usage](https://supabase.com/docs/guides/platform/database-size#disk-management).
+ - We also recommend upgrading to at least a [Large instance](https://supabase.com/docs/guides/platform/compute-add-ons) for the migration (you can downgrade later) to avoid running out of IO-Budget.
```

Comment:
```suggestion
 - We also recommend upgrading to at least a [Large instance](/docs/guides/platform/compute-add-ons) for the migration (you can downgrade later) to avoid running out of IO-Budget.
```